bluesky/plans.py

```diff
@@ -1102,24 +1307,29 @@ def scan(detectors, *args, num=None, per_step=None, md=None):
     for motor in motors:
         x_fields.extend(get_hinted_fields(motor))
 
-    default_dimensions = [(x_fields, 'primary')]
+    default_dimensions = [(x_fields, "primary")]
 
     default_hints = {}
     if len(x_fields) > 0:
         default_hints.update(dimensions=default_dimensions)
 
     # now add default_hints and override any hints from the original md (if
     # exists)
-    _md['hints'] = default_hints
-    _md['hints'].update(md.get('hints', {}) or {})
+    _md["hints"] = default_hints
+    _md["hints"].update(md.get("hints", {}) or {})
 
     full_cycler = plan_patterns.inner_product(num=num, args=args)
 
-    return (yield from scan_nd(detectors, full_cycler,
-                               per_step=per_step, md=_md))
+    return (yield from scan_nd(detectors, full_cycler, per_step=per_step, md=_md))
 
 
-def grid_scan(detectors, *args, snake_axes=None, per_step=None, md=None):
+def grid_scan(
+    detectors,
```

Comment:
Missing type.

Reply:
Good spot! Will replace based on outcome of https://github.com/bluesky/bluesky/pull/1610/files/7ebbfcf3408f15d11ece2f847a70b198103aef3d#r1310969275